src/Maker/MakeResetPassword.php

```diff
@@ -320,4 +316,86 @@ private function successMessage(InputInterface $input, ConsoleStyle $io, string
         $io->text('Then open your browser, go to "/reset-password" and enjoy!');
         $io->newLine();
     }
+
+    private function generateRequestEntity(Generator $generator, $requestClassNameDetails, $repositoryClassNameDetails): void
+    {
+        $entityClassGenerator = new EntityClassGenerator($generator, $this->doctrineHelper);
+
+        $pathRequestEntity = $entityClassGenerator->generateEntityClass($requestClassNameDetails, false);
+
+        $generator->writeChanges();
+
+        $manipulator = new ClassSourceManipulator(
+            $this->fileManager->getFileContents($pathRequestEntity)
+        );
+
+        $manipulator->addInterface(ResetPasswordRequestInterface::class);
+
+        $manipulator->addTrait(ResetPasswordRequestTrait::class);
+
+        $manipulator->addConstructor([
+            (new Param('user'))->setType('object')->getNode(),
+            (new Param('expiresAt'))->setType('\DateTimeInterface')->getNode(),
+            (new Param('selector'))->setType('string')->getNode(),
+            (new Param('hashedToken'))->setType('string')->getNode(),
+        ], <<<'CODE'
+<?php
+$this->user = $user;
+$this->initialize($expiresAt, $selector, $hashedToken);
+CODE
+        );
+        $manipulator->addProperty('id', [
+            '@ORM\Id()',
+            '@ORM\GeneratedValue()',
+            '@ORM\Column(type="integer")',
+        ]);
+
+        $manipulator->addManyToOneRelation((new RelationManyToOne())
+            ->setPropertyName('user')
+            ->setTargetClassName("App\Entity\User")
+            ->setIsNullable(true)
+            ->setMapInverseRelation(false)
+            ->setReturnType('object', false)
+            ->avoidSetter()
+        );
+
+        $this->fileManager->dumpFile($pathRequestEntity, $manipulator->getSourceCode());
+
+        $pathRequestRepository = $this->fileManager->getRelativePathForFutureClass(
+            $repositoryClassNameDetails->getFullName()
+        );
+
+        $manipulator = new ClassSourceManipulator(
+            $this->fileManager->getFileContents($pathRequestRepository)
+        );
+
+        $manipulator->clearClassNodeStmts();
+
+        $manipulator->addInterface(ResetPasswordRequestRepositoryInterface::class);
+
+        $manipulator->addTrait(ResetPasswordRequestRepositoryTrait::class);
+
+        $manipulator->addConstructor([
+            (new Param('registry'))->setType('ManagerRegistry')->getNode(),
+        ], <<<'CODE'
+<?php
+parent::__construct($registry, ResetPasswordRequest::class);
+CODE
+        );
```

Comment:
As we talked about, if we want to avoid the comments in the repository class, let's pass an argument to `EntityClassGenerator::generateEntityClass()` to make it not generate the comments. That method name is getting a little long, but that's ok - it's an internal class.